plugins/chat/assets/javascripts/discourse/models/chat-message-draft.js

```diff
@@ -0,0 +1,62 @@
+import { tracked } from "@glimmer/tracking";
+
+export default class ChatMessageDraft {
+  static create(args = {}) {
+    return new ChatMessageDraft(args ?? {});
+  }
+
+  @tracked uploads;
+  @tracked message;
+  @tracked _replyToMsg;
+
+  constructor(args = {}) {
```

Comment:
Not a huge deal, but this could be written as 

```javascript
  constructor({ message = "", uploads = [], replyToMsg } = {}) {
    this.message = message;
    this.uploads = uploads;
    this.replyToMsg = replyToMsg;
  }
```

I just remembered this from David's post about the destructuring